src/main/java/seedu/address/model/ModelManager.java

```diff
@@ -111,6 +111,20 @@ public void setTask(Task target, Task editedTask) {
         addressBook.setTask(target, editedTask);
     }
 
+//    @Override
```

Comment:
Would you mind sharing why you comment this portion out?

Reply:
Sure! I was planning on copying the implementation of setTask() within this method, as they serve similar purposes of replacing an old Task with a newly edited Task, but I realised afterwards that it is not necessary to create two new methods that serve similar purposes to the original, existing setTask(), so I commented out these methods. Will remove in a future commit!